snippets/sdn-deprecation-statement.adoc

```diff
@@ -23,5 +23,5 @@
 
 [NOTE]
 ====
-OpenShift SDN CNI is deprecated as of {product-title} 4.14. As of {product-title} 4.15, the network plugin is not an option for new installations. In a subsequent future release, the OpenShift SDN network plugin is planned to be removed and no longer supported. Red Hat will provide bug fixes and support for this feature until it is removed, but this feature will no longer receive enhancements. As an alternative to OpenShift SDN CNI, you can use OVN Kubernetes CNI instead.
+OpenShift SDN CNI is deprecated as of {product-title} 4.14. As of {product-title} 4.15, the network plugin is not an option for new installations. In a subsequent future release, the OpenShift SDN network plugin is planned to be removed and no longer supported. Red{nbsp}Hat will provide bug fixes and support for this feature until it is removed, but this feature will no longer receive enhancements. As an alternative to OpenShift SDN CNI, you can use OVN Kubernetes CNI instead.
```

Comment:
I'd suggest to not use product-title in front of versions like 4.14 or 4.15 because ROSA service is not versioned. Instead we can either reference product-title clusters with version 4.14 because version is literally how we reference OCP releases in the API/CLI. Alternatively, we just specify that product-title will follow the life cycle of SDN according to OCP; and say clusters deprecated in 4.14, can't install in 4.15, can't upgrade in 4.16, discontinued in 4.17.